### ISSUE TITLE
crash

### ISSUE THREAD

**Issue by cutdachez** [TrinityCore/ymir#1]
newest sniffer version crash at game loading

**Comment by Faq** [comment]
A screenshot would be good. Client version and language?

**Comment by flashaholic88** [comment]
Yeah same here:
https://imgur.com/WqAh1WH

**Comment by funjoker** [comment]
Not a crash. It closes because if it can’t decrypt packets, it can’t sniff.
Do you use VPN or some special network stuff ?

**Comment by flashaholic88** [comment]
No i dont use any VPN or anything else. 
9.1.0.40000 Retail worked without any problems

**Comment by xXBentRealityXx** [comment]
Can confirm the 9.1.0 40120 works, I'm using enUS locale if that matters

**Comment by mdX7** [comment]
in general you should just retry it when this happens. This may even happen multiple times in a row. We dont have a solution to fix this yet, not even a possibility to reproduce it consistently. 
For some people this happens more frequently, for others it does not happen at all.

As funjoker said this is caused due to the sniffer being not able to decrypt the data. As safe measurement it kills wow in this case.

**Comment by Nyr97** [comment]
I've had this issue before with different builds, no idea what's causing it, but re-opening the sniffer and re-opening wow made the trick for me tho.

**Comment by T1ti** [comment]
Getting this crash all the time with season of mastery
![image](https://user-images.githubusercontent.com/40864460/142075368-a8b27acc-82b0-4b18-882d-01ac9968efb8.png)

**Comment by grip244** [comment]
![image](https://user-images.githubusercontent.com/24600090/143796510-465eeac3-c078-47ec-9b3c-6599d55f6b91.png)

On closing the game, and then in dumps, the pkt file is written, but only 1kb.

**Comment by ghost** [comment]
I have 15 crashes in a row today. I haven't been able to sniff at all
![image](https://user-images.githubusercontent.com/709267/147395134-329cf599-cc0e-4f11-bab9-86d4c5a174df.png)

**Comment by grip244** [comment]
Try deleting Cache, and the WTF folder. That's what fixed my issue, but if it doesn't, it might just be the version. I know 9.1.5.40944, won't work no matter what I do.

**Comment by mdX7** [comment]
> Try deleting Cache, and the WTF folder. That's what fixed my issue, but if it doesn't, it might just be the version. I know 9.1.5.40944, won't work no matter what I do.

Cache and WTF are absolutely unrelated to this, no need to delete them.
I highly advise to ALWAYS delete Cache before starting sniffing however because you'd miss some data otherwise.

> I have 15 crashes in a row today. I haven't been able to sniff at all

I might've found something, can you throw a dumped sniff over to me @ discord?

**Comment by ghost** [comment]
> > Try deleting Cache, and the WTF folder. That's what fixed my issue, but if it doesn't, it might just be the version. I know 9.1.5.40944, won't work no matter what I do.
> 
> Cache and WTF are absolutely unrelated to this, no need to delete them. I highly advise to ALWAYS delete Cache before starting sniffing however because you'd miss some data otherwise.
> 
> > I have 15 crashes in a row today. I haven't been able to sniff at all
> 
> I might've found something, can you throw a dumped sniff over to me @ discord?

Tried. can't send to you

**Comment by malcrom** [comment]
It fails far more than it used to. I seems It is getting harder and harder for me to connect everyday.

**Comment by mdX7** [comment]
Tried some more stuff, should fix a lot of crashes, can you guys try this and report back here pls:

@malcrom [ymir_classic_tbc.zip](https://github.com/TrinityCore/ymir/files/7776684/ymir_classic_tbc.zip)

@grip244 [ymir_retail.zip](https://github.com/TrinityCore/ymir/files/7776685/ymir_retail.zip)

I cannot reproduce the crashes, so i'm kinda depending on your tests here..^^

**Comment by ghost** [comment]
Works perfectly now

**Comment by Phentora** [comment]
Hey guy. The latest release [9.2.5.44232] crashes for me. I installed Npcap, then started ymir, cleared cache in WoWfolder, then launched the game. As soon as the game launches, ymir crashes.

There is no log file created and the ymir console appears to write something but i can't make out what it says before it crashes.

I am not running any proxy or VPN on my machine; just connected to my Wi-Fi.

Edit: i tried the ymir release that @mdX7 tagged and that one doesn't crash.
**Edit: I have solved the issue. It seems my problem wasn't cause by ymir but by the way i installed Npcap. I have reinstalled Npcap and unchecked the "Restrict Npcap driver's access to Administrator only". After that, i started ymir again and it worked!** 

Hope this helps someone.
